ci/travis/install-bazel.sh

```diff
@@ -38,6 +54,10 @@ else
   rm -f "${target}"
 fi
 
+cat <<"EOF" >> "${HOME}/.bashrc"
```

Comment:
Do a grep check for this and only add it if not there

Reply:
Done